.github/workflows/plugin-build.yml

```diff
@@ -0,0 +1,207 @@
+name: Reusable Build workflow
+
+on:
+  workflow_call:
+    inputs:
+      PACKAGE_VERSION:
+        required: false
+        type: string
+        description: 'Optional. The version of the package to use.'
+        default: ""
+      PACKAGE_FILES_VERSION:
+        required: false
+        type: string
+        default: ""
+      ELEMENTOR_PRO_CORE_BRANCH:
+        required: false
+        type: string
+        description: 'The branch of the core repository to use.'
```

Comment:
I think this is not relevant anymore. 
We have custom core that replaces it.

Reply:
@davseve `ELEMENTOR_PRO_CORE_BRANCH` is important because in custom core we get a branch from input of manual dispatch so we want to be able to pass that branch to the build process